src/Turn.js

```diff
@@ -1,32 +1,34 @@
 class Turn {
-    constructor(userGuess, currentCard) {
-        this.userGuess = userGuess,
-        this.currentCard = currentCard,
-        this.guessCorrect = null;
+  constructor(userGuess, currentCard) {
+    this.userGuess = userGuess,
+    this.currentCard = currentCard,
+    this.guessCorrect = null;
+  }
+  returnGuess() {
+    return this.userGuess;
+  }
+  returnCard() {
+    return this.currentCard;
+  }
+  evaluateGuess() {
+    if (this.userGuess === this.currentCard.correctAnswer) {
+      this.guessCorrect = true;
+      return true;
+    } else {
+      this.guessCorrect = false;
+      return false;
     }
-    returnGuess() {
-        return this.userGuess;
-    }
-    returnCard() {
-        return this.currentCard;
-    }
-    evaluateGuess() {
-        if (this.userGuess === this.currentCard.correctAnswer) {
-            this.guessCorrect = true;
-            return true;
-        } else {
-            this.guessCorrect = false;
-            return false;
-        }
-        //this.userGuess === this.currentCard.correctAnswer ? true : false;
```

Comment:
Why did you decide against the ternary here? Its okay, just wondering what was behind that decision.
 My note about refactoring this evaluateGuess() method isn't asking for it to be refactored into a ternary necessarily - just to be clear.

Reply:
I think I originally tried this and it wasn't passing the test. I'll play with this method and see what I can do to refactor.